influxdb/src/main/scala/akka/stream/alpakka/influxdb/scaladsl/InfluxDBSource.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.influxdb.scaladsl
+
+import akka.NotUsed
+import akka.stream.alpakka.influxdb.InfluxDBSettings
+import akka.stream.alpakka.influxdb.impl.{InfluxDBRawSourceStage, InfluxDBSourceStage}
+import akka.stream.scaladsl.Source
+import org.influxdb.InfluxDB
+import org.influxdb.dto.{Query, QueryResult}
+
+/**
+ * Scala API.
+ */
+object InfluxDBSource {
+
+  /**
+   * Java API: creates an [[akka.stream.alpakka.influxdb.impl.InfluxDBRawSourceStage]] from a given statement.
+   */
+  def apply(influxDB: InfluxDB, query: Query): Source[QueryResult, NotUsed] =
+    Source.fromGraph(
+      new InfluxDBRawSourceStage(
+        query,
```

Comment:
In a few places, I see that we are breaking out the arguments of the constructor. This is great if the line is too long and/or we want to name the arguments to improve readability/understandability, but in this case where we don't name the arguments and where it could fit on one line, I think it would be better to collapse this to one line.

Makes the code easier to comprehend, IMO.